updating/index.adoc

```diff
@@ -0,0 +1,64 @@
+
+* xref:../updating/updating-cluster-within-minor.adoc#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-within-minor.adoc#update-changing-update-server-web_updating-cluster-within-minor[Changing the update server by using the web console]
+
+['updating-clusters-overview-update-cluster-using-cli']
+== Updating a cluster within a minor version using the command-line interface (CLI)
+xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version using the CLI]: You can update an {product-title} cluster by using the `oc`. You can the update a cluster between minor versions. For more information, see the following actions:
+
+* xref:../updating/updating-cluster-cli.adoc#update-upgrading-cli_updating-cluster-cli[Updating a cluster by using the CLI]
+* xref:../updating/updating-cluster-cli.adoc#update-changing-update-server-cli_updating-cluster-cli[Changing the update server by using the CLI]
+
+['updating-clusters-overview-update-cluster-with-rhel-compute-machines']
+== Updating a cluster that includes RHEL compute machines
+xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]:  You can update an {product-title} cluster. If your cluster contains {op-system-base-full} machines, you must update those machines. You can perform the following actions:
+
+* xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute-hooks[Optional: Adding hooks to perform Ansible tasks on RHEL machines]
+* xref:../updating/updating-cluster-rhel-compute.adoc#rhel-compute-updating-minor_updating-cluster-rhel-compute[Updating RHEL compute machines in your cluster]
+
+['updating-clusters-overview-update-restricted-network-cluster']
+== Updating a restricted network cluster
+xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: A restricted network environment is one in which your cluster nodes cannot access the internet. You can update a restricted network {product-title} cluster by using the `oc`. For more information, see the following items:
+
+* xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-mirror-host[Preparing your mirror host]
+* xref:../updating/updating-restricted-network-cluster.adoc#installation-adding-registry-pull-secret_updating-restricted-network-cluster[Configuring credentials that allow images to be mirrored]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-mirror-repository_updating-restricted-network-cluster[Mirroring the {product-title} image repository]
+* xref:../updating/updating-restricted-network-cluster.adoc#machine-health-checks-pausing_updating-restricted-network-cluster[Pausing a MachineHealthCheck resource]
```

Comment:
Refer to https://docs.openshift.com/container-platform/4.6/updating/updating-restricted-network-cluster.html, no `Pausing a MachineHealthCheck resource`, but `Creating the image signature config map`

Reply:
Addressed- Added `Creating the image signature config map`